contracts/src/starknet/rewaave/tokens/ETHstaticAToken.cairo

```diff
@@ -128,6 +110,34 @@ end
 # Externals
 #
 
+@external
+func initialize_ETHstaticAToken{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(
+    name : felt,
+    symbol : felt,
+    decimals : felt,
+    initial_supply : Uint256,
+    recipient : felt,
+    controller : felt,
+):
+    let (name_) = ERC20_name()
+    let (symbol_) = ERC20_symbol()
+    let (decimals_) = ERC20_decimals()
+
+    with_attr error_message("ETHstaticAToken already initialized"):
+        assert name_ = 0
+        assert symbol_ = 0
+        assert decimals_ = 0
+    end
+
+    ERC20_initializer(name, symbol, decimals)
+    ERC20_mint(recipient, initial_supply)
+    Ownable_initializer(controller)
+    # TODO we either need to configure the last_update here, or pause the contract
```

Comment:
which of the option below has been selected? is it to be implemented now or in a separate PR?

Reply:
It seems the comment is redundant